org-code-javabuilder/validation/src/main/java/org/code/validation/TestResultMessage.java

```diff
@@ -0,0 +1,11 @@
+package org.code.validation;
+
+import org.code.protocol.ClientMessage;
+import org.code.protocol.ClientMessageType;
+
+/** A message describing a test result directed to the client's terminal. */
+public class TestResultMessage extends ClientMessage {
```

Comment:
sorry for the late change, but it might be clearer to name this `UserTestResultMessage` to align with `UserTestOutputAdapter`. I think we can keep the enum value as `TEST_RESULT` though.